app/models/user.rb

```diff
@@ -28,7 +20,7 @@ def set_location
   # Include default devise modules. Others available are:
   # :omniauthable
   devise :database_authenticatable, :registerable, :confirmable, :timeoutable,
-         :recoverable, :rememberable, :trackable, :validatable, :lockable
```

Comment:
Ok, I understand the logic. 

Yes, please remove all the validatable stuff on the initializer (if any) and also left a comment there saying that we're doing this on the model validations.